bundles/net.tourbook.chart/src/net/tourbook/chart/ChartComponentGraph.java

```diff
@@ -826,7 +826,7 @@ private boolean canChartBeOverlapped(final ArrayList<GraphDrawingData> allGraphD
       for (final GraphDrawingData graphDrawingData : allGraphDrawingData) {
 
          final ChartType chartType = graphDrawingData.getChartType();
-
+//TODO ASk wolfgang if we can support line and bars ?
          if (chartType != ChartType.LINE && chartType != ChartType.HORIZONTAL_BAR) {
```

Comment:
@wolfgang-ch Do you know if it's possible for a graph to display bars AND lines ? is it hard work ?

Reply:
>Do you know if it's possible for a graph to display bars AND lines ? 

 This graph type is already available `net.tourbook.chart.ChartType.LINE_WITH_BARS` but it is currently not used. Maybe it was already used or tested but I can't remember where, it is possible that it was never published in a release.

>is it hard work ?

Try to use this graph type

Reply:
Ok, I will try.

Reply:
When you checkout MT 01-01-2010 there is already `net.tourbook.chart.ChartDataModel.CHART_TYPE_LINE_WITH_BARS` available but could not figure out where it was used.

Reply:
When I use `net.tourbook.chart.ChartType.LINE_WITH_BARS` , that is what the graph shows. It doesn't seem to work or maybe I missed something to make it work ?
![image](https://user-images.githubusercontent.com/25235665/77115132-60c95c80-69f3-11ea-8447-802f0604f82e.png)

Reply:
In the more than last 10 years, there were some modifications in the tour chart, so I'm not wondering about these issues as this code was not used.

Reply:
So you mean it should or should NOT work ?

Reply:
As you can see on the y-axis, 2 overlayed graphs are displayed but the y-axis is not prepared for this case. Overlayed graphs are introduced just a few years ago but `net.tourbook.chart.ChartType.LINE_WITH_BARS` is far more older and not adjusted.

So it looks that it partly works but this paint method has to be adjusted. There is also the question, how to display 2 different y-axis, one on the left and one on the right but what happens when there are 3 overlayed graphs?

Reply:
When you are planning to enhance the painting method and use SWT resources, then you have to dispose them carefully. I use the eclipse sleak tool https://www.eclipse.org/articles/swt-design-2/sleak.htm to make sure all SWT resources are correctly disposed.

Reply:
Sleak can be downloaded from https://www.eclipse.org/swt/tools.php

Reply:
> 
>  There is also the question, how to display 2 different y-axis, one on the left and one on the right but what happens when there are 3 overlayed graphs?

In the software I used before MT, this is how Y-Axis data on 3+ overlayed graphs would be displayed
![image](https://user-images.githubusercontent.com/25235665/77252841-2ee70e80-6c1c-11ea-8fc1-df93995621f2.png)

Reply:
> 
> 
> When you are planning to enhance the painting method and use SWT resources, then you have to dispose them carefully. I use the eclipse sleak tool https://www.eclipse.org/articles/swt-design-2/sleak.htm to make sure all SWT resources are correctly disposed.

I don't know this tool, I will look at it and learn how to use it. Thanks

Reply:
The tour chart in MT has a very static layout, so a variable with for multiple y-axis is not available and I think it would not be very easy to implement it. 

The current overlayed graphs show only the y-axis of the last selected graph, all other y-axis are currently hidden. Maybe the left and right axis could have different y-axis but this is propably also not very easy but less complex as a vaiable width for multiple y-axis.

Reply:
I just discovered that sleak is not working in MT anymore but it worked with Eclipse 3.x. Happily I googled the problem, fixed it in the master branch and it is working now :-)

Here you can see where an image resource was created
![mt-with-sleak](https://user-images.githubusercontent.com/1283445/77256732-55369980-6c70-11ea-9d50-a5105ccdd94c.png)

I created a separate launcher with the Sleak options
![mt-sleak-options](https://user-images.githubusercontent.com/1283445/77256734-5962b700-6c70-11ea-80fe-1a7c2f27bd9e.png)